packages/dev-scripts/examples/util.ts

```diff
@@ -139,7 +140,11 @@ export function getProjectFiles(project: Project): Files {
  */
 export function getExampleProjects(): Project[] {
   const examples: Project[] = glob
-    .globSync(path.join(dir, "../../../examples/**/*/.bnexample.json"))
+    .globSync(
```

Comment:
I suppose this is needed for windows?

Cleaner might be: `path
        .join(dir, "..","..","..","examples","**",".*","..bnexample.json")`?